sdk/maps/azure-maps-geolocation/azure/maps/geolocation/models/_models.py

```diff
@@ -0,0 +1,23 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+class IpAddressToLocation(object):
```

Comment:
I strongly feel this should simply be called `Geolocation`. This is a very small API. A single client with a single operation that returns a single object.  It is just weird that they are all inconsistent. 

This is much more consistent, imo.
`GeolocationClient.get_geolocation(ip_address: string) -> Geolocation`

We would need a very good reason to go with something else.